context/context.go

```diff
@@ -13,7 +13,7 @@ import (
 
 // cap the number of git remotes looked up, since the user might have an
 // unusually large number of git remotes
-const maxRemotesForLookup = 5
+const defaultMaxRemotesForLookup = 5
```

Comment:
```suggestion
const defaultRemotesForLookup = 5
```

Max doesn't make lot of sense anymore I don't think.